src/components/ContentSidebar/ActivitySidebar.js

```diff
@@ -387,9 +387,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         }
 
         api.getFileCollaboratorsAPI(true).getFileCollaborators(file.id, successCallback, errorCallback, {
-            params: {
```

Comment:
Should we instead change the `markerGetRequest` to unwrap `params` from `requestData`?